src/lib/auth/cloud-idp-provider.ts

```diff
@@ -0,0 +1,26 @@
+import { Provider } from 'next-auth/providers'
+
+/**
+ * A custom next-auth provider to authenticate via HashiCorp's Cloud IDP service
+ */
+const CloudIdpProvider: Provider = {
+	id: 'cloud-idp',
+	name: 'Cloud IDP',
+	type: 'oauth',
+	wellKnown:
+		__config.dev_dot.auth.idp_url + '/.well-known/openid-configuration',
+	authorization: { params: { scope: 'openid' } },
```

Comment:
Looks like the IDP team added support for the `profile` and `picture` scopes! I know Learn used an avatar, but I'm not sure if devportal needs one as well.

Reply:
Yeah! I'm actually able to access the picture and profile attributes right now (see the JWT callback in the API route), so it all seems to be coming through as expected without additional scopes here 💭

Reply:
Ah gotcha! I knew `profile` was a default scope, but I guess `picture` is as well!